modules/roles_profiles/manifests/profiles/network.pp

```diff
@@ -10,8 +10,22 @@
             include ::macos_utils::bonjour_advertisements_disabled
         }
         'Windows': {
-            include win_network::set_search_domain
-            include win_network::disable_ipv6
+
+            if $facts['custom_win_location'] == 'datacenter' {
+                include win_network::set_search_domain
+                include win_network::disable_ipv6
+            }
+            if $facts['custom_win_location'] != 'datacenter' {
```

Comment:
This should be an `else` so you aren't evaluating twice

Reply:
Makes sense. I will change it in a future patch, so not to clear out this review.